src/quo/components/utilities/social/loader.clj

```diff
@@ -0,0 +1,40 @@
+(ns quo.components.utilities.social.loader
+  (:require [clojure.core]
```

Comment:
I think we don't need to import `clojure.core`, why do we have this here?

Reply:
Done